docs/components/beacon/index.html

```diff
@@ -0,0 +1,45 @@
+---
+title: Beacon
+minver: 0.21.0
+also:
+  elements/hx-beacon: <hx-beacon>
+---
+{% extends 'component.njk' %}
+
+{% block page_header %}
+  <p>
+    A beacon is an animation that attracts a user's attention
+    to a new feature or advanced functionality in a control panel.
+  </p>
+{% endblock %}
+
+{% block content %}
+  <section>
+    <header>
+      <h2 id="single-beacon">Beacon</h2>
+      {# TODO: add section description #}
+    </header>
+    <div class="example" id="vue-beaconDemo" v-cloak>
+      <div id ="Beacon">
+        <hx-beacon v-on:click="beaconHide"></hx-beacon>
+      </div>
```

Comment:
Please update this demo to include additional context for implementors with a popover.

Reply:
Changes `have been done, please review